app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -243,6 +243,15 @@ private void initChapters(@NonNull ArrayList<Chapter> chapters) {
                     break;
                 }
             }
+        } else if (getIntent().getData() != null && getIntent().getData().getScheme().equals("https")) {
+            final String chapterId = getIntent().getData().getLastPathSegment();
+            for (Chapter c : chapters) {
```

Comment:
There is a duplicate code here. :smile: 
Can you pull the chapterId in https://github.com/gdg-x/frisbee/pull/389/files#diff-cd65f5d3a3ce575d6529f521aed523dbR247 above. And then check if it exists in chapters array in 1 for loop.